tendrl/commons/event.py

```diff
@@ -0,0 +1,10 @@
+import socket
+import json
+
+
+class Event(object):
+    def __init__(self, message, host, port):
```

Comment:
Supposed to be a UNIX socket. Refer to this comment and make the appropriate changes in both codebases: https://github.com/Tendrl/node-agent/pull/67/files#r97720923